cli/crates/cli/src/cli_input/dev.rs

```diff
@@ -0,0 +1,19 @@
+use super::ProjectRef;
+use clap::Parser;
+use std::path::PathBuf;
+
+/// Start a local development server
+#[derive(Debug, Parser)]
+pub struct DevCommand {
+    #[arg(short('r'), long("graph-ref"), help = ProjectRef::ARG_DESCRIPTION)]
+    pub(crate) graph_ref: Option<ProjectRef>,
```

Comment:
We have a `GraphRef` in the `graph-ref` crate, you might want to re-use it, wrapping it in something like `GraphRefWithAccountRef` for example

Reply:
up to you, just wanted to mention it in case you weren't aware of it^^